### PR TITLE
Remove errant brace character

### PR DESCRIPTION
**Pull request by lowellrex**
Before | After
--- | ---
image | image

### CODE REVIEW

File: client/app/queue/OrganizationQueueLoadingScreen.jsx [hunk 1]
```diff
@@ -24,10 +24,7 @@ class OrganizationQueueLoadingScreen extends React.PureComponent {
     </div>;
 
     const loadingDataDisplay = <LoadingDataDisplay
-      createLoadPromise={() => {
-        return createOrgQueueLoadPromise(this.props, this.props.urlToLoad);
-      }
-      }
+      createLoadPromise={() => createOrgQueueLoadPromise(this.props, this.props.urlToLoad)}
```
Comment (lowellrex) [inline client/app/queue/OrganizationQueueLoadingScreen.jsx]: Unrelated to the offending curly brace character, but I'm working in this file for another PR so I figured I'd simplify this function call while I was at it.

File: client/app/queue/OrganizationQueue.jsx [hunk 2]
```diff
@@ -190,12 +190,11 @@ const TaskTableWithUserColumnTab = ({ description, tasks, organizationName, user
...
   }
 
-  return (<React.Fragment>
+  return <React.Fragment>
     <p className="cf-margin-top-0">{description}</p>
     <TaskTable
       customColumns={columns}
       tasks={tasks}
     />
-    }
```
Comment (lowellrex) [inline client/app/queue/OrganizationQueue.jsx]: This is the character that had to go!

Comment (kevmo) [reply client/app/queue/OrganizationQueue.jsx]: Deeply troubled character.

File: client/app/queue/OrganizationQueueLoadingScreen.jsx (hunk 1 above)
Comment (kevmo) [reply client/app/queue/OrganizationQueueLoadingScreen.jsx]: 🔪

**Pull request merged**